packages/core/upload/admin/src/hooks/__mocks__/useEditFolder.js

```diff
@@ -0,0 +1,4 @@
+export const useEditFolder = jest.fn().mockReturnValue({
+  editFolder: jest.fn().mockResolvedValue({}),
```

Comment:
Since this is a very general mock now, could you please return a realistic folder object instead, so that we test a real-world scenario here?

Reply:
sure - I would prefer to add tests in a future PR though, I would like to prioritize the ML folder fix we have to do before the release, WDYT?

Reply:
Fine with me!